report/renderer/category-renderer.js

```diff
@@ -319,7 +319,11 @@ export class CategoryRenderer {
   renderScoreGauge(category, groupDefinitions) { // eslint-disable-line no-unused-vars
     const tmpl = this.dom.createComponent('gauge');
     const wrapper = this.dom.find('a.lh-gauge__wrapper', tmpl);
-    this.dom.safelySetHref(wrapper, `#${category.id}`);
+    const url = new URL(window.location.href);
+    url.hash = category.id;
+    this.dom.safelySetHref(wrapper, url.href);
+    // wrapper.href = url.href;
```

Comment:
if the modification in `safelySetHref` is not desirable, we could do one of these ... thoughts ?

Reply:
(TODO: add comment about why fully qualified link is needed)

Reply:
fyi `wrapper.addEventListener('click', () => window.location.hash = category.id);` would work too ... but is that bad for a11y?